client/layout/masterbar/empty.jsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+/**
+ * Style dependencies
+ */
+
+const EmptyMasterbar = ( { className } ) => (
+	<header id="header" className={ classNames( 'masterbar', className ) } />
```

Comment:
I think the `className` prop is unnecessary - [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it). If we don't use it, let's ditch it.

Reply:
Good call. Removed.